src/fdwave_raja/target_3d.cpp

```diff
@@ -167,6 +167,43 @@ void compute_pml_3d(RAJA::Index_type const x3, const RAJA::Index_type x4, const
                                      *(uViewConst(i+lx,j+ly,k+1+lz)-uViewConst(i+lx,j+ly,k-1+lz))*hdz_2 ) )
                                  / (1.f+etaViewConst(i+1,j+1,k+1));
                              });
+#else
+  for (llint i = x3; i < x4; ++i) {
```

Comment:
Same loop body as above but with for-loops and it gets the same poor performance.